File: Array/linchuanxin/Algorithm/day26_0626_删除节点/day26_leetcode237.c
//
// Created by LCX on 2020/6/27.
//


  struct ListNode {
      int val;
      struct ListNode *next;
  };

void deleteNode(struct ListNode* node) {
    node->val = node->next->val;
    node->next = node->next->next;
}

File: Array/linchuanxin/Algorithm/day26_0626_删除节点/day26_leetcode237.kt
package day26

import sun.rmi.runtime.Log
import java.util.ArrayList



 class ListNode(var `val`: Int) {
     var next: ListNode? = null
 }

class Solution {
    fun deleteNode(node: ListNode?) {
        node?.`val` = node?.next?.`val`!!
        node.next = node.next!!.next
    }
}

